src/index.css

```
@@ -40,6 +42,675 @@ button {
+    text-decoration: none;
+    line-height: 1;
+    transition: color var(--dur-fast) var(--ease-out);
+}
+.chomp-nav-link::after {
+    content: "";
+    position: absolute;
+    left: 10px;
+    right: 10px;
+    bottom: -2px;
+    height: 1.5px;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)